### PR TITLE
Improve Convolution Gradient() Method

### PR DESCRIPTION
**Pull request by ShikharJ**
I'll need to test for the outputs of the GANs. Then this can be merged.

### CODE REVIEW

**Comment by ShikharJ** [comment]
@zoq The output looks fine, so this is good to go in my opinion. The test failure is unrelated.

**Comment by ShikharJ** [comment]
Ping @zoq.

File: src/mlpack/methods/ann/layer/reparametrization.hpp [hunk 1]
```diff
@@ -130,15 +130,15 @@ class Reparametrization
...
-  double beta;
-
   //! If false, sample will be constant.
   bool stochastic;
 
   //! If false, KL error will not be included in Backward function.
   bool includeKl;
 
+  //! The beta hyperparameter for constrained variational frameworks.
+  double beta;
```
Comment (zoq) [inline src/mlpack/methods/ann/layer/reparametrization.hpp]: Nice catch thanks!

**Comment by zoq** [comment]
Nice improvement, thanks!

**Pull request merged**